.editorconfig

```diff
@@ -9,6 +9,12 @@ trim_trailing_whitespace = true
 indent_style = space
 indent_size = 2
 
-[{Makefile,go.mod,go.sum,*.go}]
+[{Makefile,go.mod,go.sum,*.go,.gitmodules}]
 indent_style = tab
 indent_size = 8
+
+[{*.md,Dockerfile}]
+indent_size = 4
+trim_trailing_whitespace = false
```

Comment:
I'm curious about the addition of this section.

Reply:
Trailing whitespace has meaning in markdown files, so I guess that's why this was introduced.
Two spaces at the end of a line force a line break.

Reply:
@mstruebing indeed. This isn't required for the Dockerfile though... I'll fix that.